com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -37,7 +37,7 @@ public static class NetworkSpawnManager
         /// </summary>
         /// <param name="position">The position to spawn the object at</param>
         /// <param name="rotation">The rotation to spawn the object with</param>
-        public delegate NetworkObject SpawnHandlerDelegate(Vector3 position, Quaternion rotation);
+        public delegate NetworkObject SpawnHandlerDelegate(ulong clientId, Vector3 position, Quaternion rotation);
```

Comment:
Shouldn't this be called `OwnerId` because that's what this value stands for right? It is the future owner of the spawned object. Please also add a parameter documentation

Reply:
or rather `ownerClientId` :P

Reply:
fixed it, just saw your suggestions above 😄